python/ray/tune/tests/test_trial_runner.py

```diff
@@ -2051,13 +2053,14 @@ def testPauseThenResume(self):
         runner = TrialRunner()
         kwargs = {
             "stopping_criterion": {
-                "training_iteration": 2
+                "training_iteration": 3
             },
             "resources": Resources(cpu=1, gpu=1),
         }
         runner.add_trial(Trial("__fake", **kwargs))
         trials = runner.get_trials()
 
+        runner.step()
```

Comment:
Why make all of these modifications in this file? The tests are written so that the number of operations on the runner matter for correctness.

Reply:
(specifically referring to the additional `runner.step`)

Reply:
imo the tests are wrong as they're currently written but let me double check

Reply:
okay i reverted the tests, but i think it's still odd to test the specific edge case where we call save before the trial even finishes a single iteration (meaning there's no result to evaluate the checkpoint on). nevertheless modified the code to pass the original tests